Draft Location Services WN

bobjacobsen opened the pull request:
This is the first "official" draft of a new Location Services Working Note.  It's been through multiple rounds of discussion on the OpenLCB list. All comments have been addressed.

bobjacobsen commented:
It would be nice to get this merged so I can link it from the [index page](https://openlcb.org/openlcb-and-lcc-documents/openlcb-specifications/).

atanisoft commented on drafts/generated/LocationServiceWN.txt:
```diff
@@ -0,0 +1,625 @@
+   Note that 8-bit IEEE does not have a suitable resolution at a suitable maximum range. 16-bit IEEE
+   provides roughly 16mm steps at +/-25m
+
+   0x08 Analog information – for reporting analog information (voltage, current, temperature, water level,
```
was 0x07 intentionally skipped?

bobjacobsen replied:
Thanks for the careful reading.  0x07 was originally proposed for JSON formatted data.  When that was dropped, the values weren't renumbered as there was some prototype code that was already using 0x08.  There's not likely to be any other code doing that, though, so it might be better to just renumber these.

atanisoft commented on drafts/generated/LocationServiceWN.txt:
```diff
@@ -0,0 +1,625 @@
+
+   9: Meters3
+
+   10: Meters/second
```
is there a reason for decimal notation on these compared to hex for all other type bytes?

bobjacobsen replied:
The hex notation for the standard block type is a historical accident:  Originally, that was just a 6-bit field.  That's now a full byte, so I'll change those back to decimal numbers.  Thanks for noticing that!

bobjacobsen commented:
Marking as WIP as discussed has diverged somewhat on the OpenLCB groups.io list.

bobjacobsen commented:
For more background, see this [OpenLCB groups.io topic](https://groups.io/g/openlcb/topic/106188174).

bobjacobsen commented:
I believe I've addressed all comments on these documents and achieved a reasonable consensus on the list.  Could we please review and merge this PR?

bobjacobsen commented:
This PR has been sitting for a while. It creates a Working Note, not even a draft Standard or TN.   AFAIK, I've addressed all the comments from several times it's been discussed.  I'd like to merge it so it can be easily linked from the web.

If I don't hear objections over the next couple days, I'll merge this early next week.
